lib/kubernetes-deploy/runner_task_config_validator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class RunnerTaskConfigValidator < TaskConfigValidator
+    attr_accessor :template, :args
```

Comment:
I'm curious about why you chose to make these  `attr_accessor` vs. args to `initialize`?

Reply:
I don't really have a good answer for that 🙂 and it _does_ look better, so I just sent a commit changing it. Thanks!